### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by meganrm**
Problem: 
We need to document the schema well for people making datasets and for our own understanding of the database. 

Solution:
I used [wetzel](https://nicedoc.io/CesiumGS/wetzel) to auto generate docs in a Makefile, and then ASCII doctor to convert to html files, which then get pushed to [gh-pages as our documentation](https://allen-cell-animated.github.io/cell-feature-data/). 

NOTE: Wetzel was missing some formatting for certain data structures. I included the package here for development, but I've forked their project and plan to add a cleaner version of these changes to their project so we can install it from 'npm'. But for now you can ignore everything in `wetzel`.

### CODE REVIEW

File: data-validation/data-prep.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const _ = require("lodash");
+
+const validate = (data, schema) => {
+    let dataCopy;
+    if (data.length !== undefined) {
+        dataCopy = [...data]
+    } else {
+        dataCopy = {...data}
+    }
```
Comment (schoinh) [inline data-validation/data-prep.js:9]: Hm.... so how come we don't need to do the initialize part (setting values to empty strings, 0s, etc.) anymore?

**Review by schoinh — COMMENTED** [review]
This is super awesome. I did wish there were an easy way to provide some kind of side nav (a table of contents of sorts) but doesn't look like it? Just a couple little comments.

**Comment by meganrm** [comment]
> 

I think there is, or a couple different options. I didn't like the default table of contents at the top, so I was going to look into it more

File: data-validation/data-prep.js (hunk 1 above)
Comment (meganrm) [reply data-validation/data-prep.js:9]: the "coerce types" setting in initializing ajv does this. https://github.com/allen-cell-animated/cell-feature-data/pull/36/files

**Review by schoinh — APPROVED** [review]

**Review by toloudis — APPROVED** [review]

**Pull request merged**